app/scripts/controllers/app-state.js

```diff
@@ -74,6 +75,9 @@ export default class AppStateController extends EventEmitter {
 
     const { preferences } = preferencesStore.getState();
     this._setInactiveTimeout(preferences.autoLockTimeLimit);
+
+    this._messenger = messenger;
```

Comment:
I think we should try and match `BaseControllerV2` for consistency sake, so the constructor argument stays as `messenger` but the internal property would be called `messagingSystem`.